test/organizationPurposes.spec.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
why does this test have like a random name? why not `test/organization_types_by_purpose.spec.js`?

Reply:
huh yeah, that was a bad name. renamed.